src/components/common.js

```diff
@@ -307,7 +317,8 @@ const BaseSelect = ({ value, newOptions, id, findValue, ...props }) => {
     ...commonSelectProps,
     getOptionLabel: ({ value, label }) => label || value.toString(),
     value: newValue || null, // need null instead of undefined to clear the select
-    options: newOptions
+    options: newOptions,
+    formatGroupLabel: groupedLabels => formatGroupLabel(groupedLabels)
```

Comment:
nit: the function already has the right signature and a matching name, so this could be:
```suggestion
    formatGroupLabel
```